content/en/blog/2022/otel-demo-app-nomad/index.md

```diff
@@ -0,0 +1,383 @@
+---
+title: Running the OpenTelemetry Demo App on HashiCorp Nomad
+linkTitle: OTel Demo App on Nomad
+date: 2022-12-12
+author: >-
+  [Adriana Villela](https://github.com/avillela) (Lightstep)
+draft: true
```

Comment:
You can safely remove that, it only will cause problems if we publish and it's still there